crm_lead_firstname/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* Rafael Blasco <rafabn@antiun.com>
```

Comment:
Don't put these old emails. Better to not put any mail in this case.

Reply:
What should be updated can you insert a code suggestion?
I don't know what I need to change...

Reply:
```suggestion
* Rafael Blasco
```